.github/workflows/pr.yaml

```diff
@@ -127,12 +128,14 @@ jobs:
         key: pip_cache_py_${{ matrix.python-version }}_gxy_${{ needs.setup.outputs.galaxy-head-sha }}
     - name: Set fail level for pull request
       if: ${{ github.event_name == 'pull_request' }}
-      run:
+      run: |
         echo "FAIL_LEVEL=warn" >> "$GITHUB_ENV"
+        echo "SKIP=-s tool_xsd" >> "$GITHUB_ENV"
     - name: Set fail level for merge
       if: ${{ github.event_name != 'pull_request' }}
       run:
```

Comment:
Guess its the missing `|` here (should be like on line 131). Can you change this?